.gitignore

```diff
@@ -4,3 +4,4 @@ __pycache__/
 /dist/
 /.coverage
 /htmlcov/
+*.swp
```

Comment:
I think this is fine because `vim` is so ubiquitous. But I recommend [a global ignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) or [using local exclude](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file) that way you can solve this for multiple repos at once or for custom ignored files.

Reply:
Also, you could change the `swp` file directory in your vimrc, it will annoy you on every project:

```
set backupdir=/tmp//
set directory=/tmp//
set undodir=/tmp//
```

Reply:
I'm really used to add *.swp to the gitignore. Basically all of my projects have it.

Reply:
Should I remove it or can I keep in this PR?

Reply:
Like, I said, I think this is fine. This recommendation is mostly because `*.swp` is very common but not _universal_ so many other projects might not have it and it might be good to have a local global ignore.

Reply:
For curiosity I checked what the linux kernel does, and they [just ignore `~*` files](https://github.com/torvalds/linux/blob/master/.gitignore#L133), which I think vim also uses and may be more generic.

Reply:
Changed to `~*`. https://github.com/HathorNetwork/python-hathorlib/pull/6/commits/fd6528e49ed8e3c4f09bb1fc9e0d8dd0eccb5db4